.github/workflows/process_new_pull_request.yml

```diff
@@ -0,0 +1,112 @@
+name: Process New Pull Request
+
+on:
+  pull_request:
+    types: [opened]
+  merge_group:
+    types: [checks_requested]
+  issues:
+    types:
+      - labeled
+  pull_request_target:
+    branches:
+      - develop
+      - release-*
+
+permissions: read-all
+
+jobs:
+  verify_cla:
+    name: Verify CLA
+    runs-on: ubuntu-22.04
+    permissions:
+      issues: write
+      pull-requests: write
```

Comment:
After putting the oppiabot stuff back in oppiabot.yml, let's change this to just write permission on PRs.